src/components/Footer/Footer.tsx

```diff
@@ -23,39 +36,39 @@ export const Footer = ({ screenWidth = "desktop" }: Props) => {
           isMobile ? "flex-col !gap-[26px]" : "flex-row"
         }`}
       >
-        <div
-          className={`flex flex-col gap-[40px] sm:gap-[26px] ${
-            isMobile ? "gap-[26px]" : "gap-[40px]"
-          }`}
-        >
-          <div className="flex flex-col gap-[24px] ">
-            <p
-              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
-                isMobile ? "text-center" : "text-left"
-              }`}
-            >
-              Expert Event Solutions
-            </p>
-            <a
-              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
-                isMobile ? "text-center" : "text-left"
-              }`}
-              onClick={() => {}}
-            >
-              260 Daytona Blvd, Daytona <br /> Beach, Florida 32114, United{" "}
-              <br /> States
-            </a>
-            <a
-              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
-                isMobile ? "text-center" : "text-left"
-              }`}
-              onClick={() => {}}
-              href="mailto:Info@experteventsolutions.com"
-            >
-              Info@experteventsolutions.com
-            </a>
+        {companyContactInfo && companyContactInfo[0] ? (
+          <div
+            className={`flex flex-col gap-[40px] sm:gap-[26px] ${
+              isMobile ? "gap-[26px]" : "gap-[40px]"
+            }`}
+          >
+            <div className="flex flex-col gap-[24px] ">
+              <p
+                className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
+                  isMobile ? "text-center" : "text-left"
+                }`}
+              >
+                {companyContactInfo ? companyContactInfo[0]?.companyName : ""}
+              </p>
+              <a
+                className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer w-[271px] ${
+                  isMobile ? "text-center" : "text-left"
+                }`}
+                onClick={() => console.log("clicked address")}
+              >
+                {companyContactInfo ? companyContactInfo[0]?.address : ""}
+              </a>
+              <p
+                className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
+                  isMobile ? "text-center" : "text-left"
+                }`}
+                onClick={() => console.log("clicked email")}
```

Comment:
Please compare with Main and see if this matches the footer design, because for the footer we removed all the console.log statements, and they are here now.